net.shrew.ike.qikea.json

```diff
@@ -0,0 +1,101 @@
+{
+    "app-id": "net.shrew.ike.qikea",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
```

Comment:
Any reason you are such an old runtime? The latest is 20.08.

Reply:
Yes, because I need an old version of openssl. The maximum compatible version is 1.0.2

Reply:
I think it would be better to use 20.08 and bundle openssl 1.0 in the app.

Reply:
Ok, I am working about it

Reply:
Done with c45464f